app/models/runs_extractors.rb

```diff
@@ -75,8 +75,12 @@ def extract(classification, and_reduce: false)
           ReduceWorker
         end
 
-        worker.set(queue: workflow.custom_queue_name) if workflow.custom_queue_name.present?
-        worker.perform_async(classification.workflow_id, 'Workflow', classification.subject_id, classification.user_id, ids)
+        if workflow.custom_queue_name.present?
+          worker.set(queue: workflow.custom_queue_name)
```

Comment:
again - i think you can replace the worker with the `Sidekiq::Worker::Setter` instance and use the `perform_async` only once.